tools/python_api/requirements_dev.txt

```diff
@@ -3,3 +3,4 @@ pytest
 pandas
 networkx~=3.0.0
 numpy
+pyarrow
```

Comment:
We should probably specify a version here in case the compatibility breaks.

Reply:
Done.